src/Components/v2/Carousel.tsx

```diff
@@ -202,12 +206,16 @@ export class BaseCarousel extends React.Component<
 
   /**
    * Since Flickity doesn't support SSR rendering, we need to load it once the
-   * client has mounted. During the server-side pass we use a Flex wrapper instead.
+   * client has mounted. During the server-side pass we use a Flex wrapper.
+   *
+   * In fact, the flickity library can't even be loaded in environments that do
+   * not have a global `window` object. Thus we need to lazy load it here with
+   * an inline `require` statement.
    */
   componentDidMount() {
     const { setCarouselRef } = this.props
-    const Flickity = require("flickity")
 
+    const Flickity = require("flickity") as typeof FlickityType
```

Comment:
Because using `require("Flickity")` doesn't provide the class typings, we cast it here instead.